apps/flux-system/sbox/workload-identity/workload-identity-federated-credential.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managedidentity.azure.com/v1api20220131preview
+kind: FederatedIdentityCredential
+metadata:
+  name: ${WI_NAME}-${WI_CLUSTER}-fic
+  namespace: ${NAMESPACE}
+spec:
+  owner:
+    name: aks-sbox-mi
```

Comment:
same here with owner and variable if possible -- just so we know we can take similar approach in all envs